scripts/create-kubestellar-demo-env.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Deploys Kubestellar Environment
+
+set -e
+
+echo -e "Checking that pre-req softwares are installed..."
+curl -s -o check_pre_req.sh https://raw.githubusercontent.com/kubestellar/kubestellar/v0.23.1/hack/check_pre_req.sh
+chmod +x check_pre_req.sh
+./check_pre_req.sh -V kflex ocm helm kubectl docker kind
+
+output=$(./check_pre_req.sh -V kflex ocm helm kubectl docker kind)
```

Comment:
I now see two identical invocations of the check_pre_req.sh script, one with the output going to stdout and one with the output going to a shell variable. FYI, you could just do the latter and echo the contents of the shell variable.